### ISSUE TITLE
Honeywell Wi-Fi Thermostat Support

### ISSUE THREAD

**Issue by mrstegeman** [mozilla-iot/addon-list#674]
As a user, I'd like to control my Honeywell Wi-Fi thermostat via the WebThings Gateway.

**Comment by kevin-garcia** [comment]
I'm in the same boat as @mrstegeman, with a Honeywell Wi-Fi thermostat I'd like to control through the gateway. Just adding a +1 to the request. 

I am not a developer, so while I'd love to attempt building an add-on, I do not feel like I have the skills to tackle that.

**Comment by mrstegeman** [comment]
@kevin-garcia what model thermostat do you have?

**Comment by kevin-garcia** [comment]
@mrstegeman I have the T5+.

**Comment by mrstegeman** [comment]
Does it use the Total Connect Comfort service/app?

**Comment by kevin-garcia** [comment]
No, it uses the Honeywell Home app. I registered an account with Total Connect Comfort to see if it could was supported, but the T5+ wasn't one of the supported devices.

**Comment by mrstegeman** [comment]
Ah, ok, I was looking at Home Assistant's integrations, and even they only support the Total Connect Comfort devices.
